apps/perf-test/src/scenarioRenderTypes.js

```diff
@@ -0,0 +1,22 @@
+/**
+ * You don't have to add scenarios to this structure unless
+ * you want their render types to differ from the default (mount only).
+ *
+ * Note:
+ * You should not need to have re-render tests in most cases because mount test provides enough coverage.
+ * It is mostly usefual for cases where component has memoization logics. And in case of re-rendering,
+ * memoization logic help avoid certain code paths.
+ */
+
+const AllRenderTypes = ['Mount', 'VirtualRerender'];
+const DefaultRenderTypes = ['Mount'];
```

Comment:
nit: The casing is weird for string types, I think it was better before. `mount` and `virtual-rerender` would be fine and consistent.

Reply:
I would also advise making a `RenderType`, like `type RenderType = 'mount' | 'virtual-rerender';`. `AllRenderTypes` and `DefaultRenderTypes` would then be `RenderType[]`. The definition for `RenderType` is where I thought we would have an explanation of what `virtual-rerender` is and means.